src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -160,6 +160,7 @@ const SampleChatInfoScreen = () => {
   const {channelInfo, goBack, onContactPressed} = useAnonymousChatInfoScreenHook();
 
   const [isLoadingMembers] = React.useState<boolean>(false);
+  // TODO: Change this into useUserAuthHook
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The TODO comment indicates that `useProfileHook` should be replaced with `useUserAuthHook`. However, this change has not been made. To maintain consistency and ensure the new authentication system works as expected, it's important to replace `useProfileHook` with `useUserAuthHook`.
```diff
-   const {signedProfileId} = useProfileHook();
+   const {signedProfileId} = useUserAuthHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->